src/pages/components/select/usage.mdx

```diff
@@ -141,59 +178,186 @@ Inline selects have less visual weight on a page because they are borderless.
+| _Focus_     | When a user tabs to or clicks on the select field, the field becomes focused, indicating the user has successfully navigated to the component.                                                                                                               |
+| _Error_     | When the required select value has not been selected. It can also be triggered due to a system error. This state requires a user response before data can be submitted or saved.                                                                             |
+| _Warning_   | When you need to call the user's attention to an exception condition. The condition might not be an error but can cause problems if not resolved.                                                                                                            |
+| _Disabled_  | When the user is not allowed to interact with the select due to either permissions, dependencies, or prerequisites. The disabled state completely removes the interactive function from a component. The styling is not subject to WCAG contrast compliance. |
+| _Skeleton_  | Used on an initial page load to indicate that the number input has not yet fully loaded.                                                                                                                                                                     |
+| _Read-only_ | _Coming soon!_                                                                                                                                                                                                                                               |
+
+<Row>
+<Column colLg={12}>
+
+![Select interactive states](images/select-usage-9.png)
+
+</Column>
+</Row>
+
+### Interactions
+
+#### Mouse
+
+Users trigger a dropdown menu to open by clicking the chevron icon or clicking
+anywhere within the field. Users can close the menu by clicking the chevron icon
+or clicking outside of the menu. To select an option the user can click anywhere
+inside an option container.
+
+<Row>
+<Column colLg={8}>
+
+![Select mouse interaction](images/select-usage-10.png)
 
 </Column>
 </Row>
 
-## Best practices
+#### Keyboard
 
-### Labels
+- The dropdown field is the element that receives focus. All keyboard
+  interactions happen from this element.
+- Users can open the dropdown menu by pressing `Space`, `Enter`, `Down arrow` or
+  the `Up arrow`.
+- Users can move the highlighted option to the next option by pressing the
+  `Down arrow`.
+- Users can move the highlighted option to the previous option by pressing the
+  `Up arrow`.
+- Users can close the dropdown menu by pressing `Escape`, `Space`, or `Enter`.
 
-Labels are essential to the usability of forms. Do not place a label inside a
-_select_ element. Use sentence case and no more than three words.
+<Row>
+<Column colLg={8}>
+
+![Select keyboard interaction](images/select-usage-11.png)
```

Comment:
I notice in the second part of this image you have the pink overlay on the field input, I don't think it's needed in this image.